HubNotificationHandler/src/main/java/com/eagle/hubnotifier/service/impl/NotifyHookServiceImpl.java

```diff
@@ -4,17 +4,15 @@
 import com.eagle.hubnotifier.constants.Constants;
 import com.eagle.hubnotifier.model.*;
 import com.eagle.hubnotifier.producer.NotifyHookProducer;
-import com.eagle.hubnotifier.repository.HubPostRepository;
-import com.eagle.hubnotifier.repository.HubTopicRepository;
-import com.eagle.hubnotifier.repository.HubUserRepository;
-import com.eagle.hubnotifier.repository.TopicFollowerRepository;
+import com.eagle.hubnotifier.repository.*;
```

Comment:
Need to optimise this class more.
1. create NotificationEvent in handleNotifyKafkaTopicRequest method and pass the object to each methods... 
2. From handleNotifyKafkaTopicRequest call notifyHandler.sendNotification
3. see if any other improvement can be done